src/components/Profile/Profile.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMissions } from '../../redux/Mission/missionSlice';
+
+function Profile() {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.mission.data);
+  const [mission, setMission] = useState([]);
+  const [empty, setEmpty] = useState(false);
+  useEffect(() => {
+    dispatch(getMissions());
```

Comment:
`getMissions()` is a thunk and it gets data from the API. It should only be called once, and that too in the `Missions` component. Try removing this from `Profile` component.

Reply:
Thank you. getMissions( has been removed from Profile